packages/application-shell/src/components/login-sso-callback/login-sso-callback.js

```diff
@@ -32,14 +31,15 @@ export class LoginSSOCallback extends React.PureComponent {
     location: PropTypes.shape({
       hash: PropTypes.string.isRequired,
     }).isRequired,
-    redirectTo: PropTypes.func.isRequired,
     requestAccessToken: PropTypes.func.isRequired,
   };
 
   state = {
     hasAuthenticationFailed: false,
   };
 
+  redirectTo = targetUrl => window.location.replace(targetUrl);
```

Comment:
this is failing the tests, is there any reason why you did this?

Reply:
jsdom doesn't implement window.location.replace

Reply:
Ah I’ll fix that no worries